src/utils/googleAnalytics.ts

```diff
@@ -1,47 +1,80 @@
 import { useCallback, useEffect, useState } from 'react'
-import ReactGA, { EventArgs } from 'react-ga'
-import { getNetworkInfo } from 'src/config'
+import ReactGA, { FieldsObject } from 'react-ga'
 
+import { getCurrentEnvironment, getNetworkInfo } from 'src/config'
 import { getGoogleAnalyticsTrackingID } from 'src/config'
+import { NetworkSettings } from 'src/config/networks/network'
 import { COOKIES_KEY } from 'src/logic/cookies/model/cookie'
 import { loadFromCookie, removeCookie } from 'src/logic/cookies/utils'
+import { IS_PRODUCTION } from './constants'
+
+const IS_STAGING = getCurrentEnvironment() === 'staging'
+const SHOULD_TRACK_ANALYTICS = IS_PRODUCTION || IS_STAGING
 
 export const SAFE_NAVIGATION_EVENT = 'Safe Navigation'
 
-export const COOKIES_LIST = [
-  { name: '_ga', path: '/' },
-  { name: '_gat', path: '/' },
-  { name: '_gid', path: '/' },
-]
+type TrackAnalyticsEventEvent = {
+  eventCategory: string
```

Comment:
Why prepend each prop with `event`?

Reply:
It's not possible to send custom vars with `ReactGA.event()` (which [prepends](https://github.com/react-ga/react-ga/blob/984a29c61d393eda13058a9e62e0e42ed170fb6e/src/core.js#L323) `event` internally), so we have to call GA directly with `ReactGA.ga()` which [takes](https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#events) `event...`.

Reply:
Do I get it right that on the GA dashboard side, nothing changes except that we add a new chain param?

Reply:
Yes